spring-kafka/src/main/java/org/springframework/kafka/support/serializer/JsonWrapperException.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2015-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.kafka.support.serializer;
+
+import org.apache.kafka.common.KafkaException;
+
+/**
+ * Specific wrapper, extending {@link KafkaException}.
+ *
+ * @author Igor Stepanov
+ */
+public class JsonWrapperException extends KafkaException {
```

Comment:
Not sure about this - maybe I should use the already defined project-specific [KafkaException](https://github.com/spring-projects/spring-kafka/blob/master/spring-kafka/src/main/java/org/springframework/kafka/KafkaException.java).

Pros:
 - it's not good to define a new Exception for each package;
Cons:
 - my exception extends KafkaException from 'kafka-client' which is good for such a low-level functionality.